playbooks/templates/rax-maas/swift_account_replication_check.yaml.j2

```diff
@@ -8,7 +8,7 @@ timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeou
 disabled    : "{{ (inventory_hostname not in maas_swift_recon_nodes or check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
 details     :
     file    : run_plugin_in_venv.sh
-    args    : ["{{ maas_plugin_dir }}/swift-recon.py", "--swift-recon-path", "{{ swift_recon_path }}", "--ring-type", "account", "replication"]
+    args    : ["{{ maas_plugin_dir }}/swift-recon.py", {% if deploy_osp %}"--deploy_osp",{% endif %}"--swift-recon-path", "{{ swift_recon_path }}", "--ring-type", "account", "replication"]
```

Comment:
Should be changed to 

> {% if deploy_osp | default(False) %}

Instead of {% if deploy_osp %}

Gates are breaking because of it....